src/BaseModule.php

```diff
@@ -211,7 +223,7 @@ public function run(array $request = []): ResponseInterface
 
 		$this->profiler->set(microtime(true) - $timestamp, 'init');
 
-		switch ($this->server['REQUEST_METHOD'] ?? Router::GET) {
+		switch ($this->args->getMethod() ?? Router::GET) {
```

Comment:
Doesn't `getMethod()` should always return a value? So we don't need that `?? ..` here anymore, I think.